app/src/backend/mocks/createCollection.ts

```diff
@@ -0,0 +1,55 @@
+import firebaseApp from "../../firebase";
+import { getFirestore, Timestamp, updateDoc, addDoc, collection } from "firebase/firestore";
```

Comment:
![3e73f2cd94268663f977b0cb99ac1a00dc9638354ea8fe4140da4790b3c285c8](https://github.com/requestly/requestly/assets/16779465/7c8834ba-6adf-4053-8e25-e894f9e6cc63)
Old mocks are breaking the styling. Let's comments out the old mocks fetching code as we have deprecated them anyway